doc/sphinx/README.rst

```diff
@@ -0,0 +1,298 @@
+
+          Print nat.
+          Definition a := 1.
+
+    Here is a list of permissible options:
+
+    - Display options
+
+      - ``all``: Display input and output
+      - ``in``: Display only input
+      - ``out``: Display only output
+      - ``none``: Display neither (useful for setup commands)
+
+    - Behavior options
+
+      - ``reset``: Send a ``Reset Initial`` command before running this block
+      - ``undo``: Send an ``Undo n`` (``n`` = number of sentences) command after
+        running all the commands in this block
+
+``.. coqdoc::`` A reST directive to display Coqtop-formatted source code.
+    Usage::
+
+       .. coqdoc::
+
+          Coq code to highlight
+
+    Example::
+
+       .. coqdoc::
+
+          Definition test := 1.
+
+``.. example::`` A reST directive for examples.
+    This behaves like a generic admonition; see
+    http://docutils.sourceforge.net/docs/ref/rst/directives.html#generic-admonition
+    for more details.
+
+    Example::
+
+       .. example:: Adding a hint to a database
+
+          The following adds ``plus_comm`` to the ``plu`` database:
+
+          .. coqdoc::
+
+             Hint Resolve plus_comm : plu.
+
+``.. inference::`` A reST directive to format inference rules.
+    This also serves as a small illustration of the way to create new Sphinx
+    directives.
+
+    Usage::
+
+       .. inference:: name
+
+          newline-separated premisses
+          ------------------------
+          conclusion
+
+    Example::
+
+       .. inference:: Prod-Pro
+
+          \WTEG{T}{s}
+          s \in \Sort
+          \WTE{\Gamma::(x:T)}{U}{\Prop}
+          -----------------------------
+          \WTEG{\forall~x:T,U}{\Prop}
+
+``.. preamble::`` A reST directive for hidden math.
+    Mostly useful to let MathJax know about `\def`\ s and `\newcommand`\ s.
+
+    Example::
+
+       .. preamble::
+
+          \newcommand{\paren}[#1]{\left(#1\right)}
+
+Coq roles
+=========
+
+In addition to the objects and directives above, the ``coqrst`` Sphinx plugin defines the following roles:
+
+``:g:`` Coq code.
+    Use this for Gallina and Ltac snippets::
+
+       :g:`apply plus_comm; reflexivity`
+       :g:`Set Printing All.`
+       :g:`forall (x: t), P(x)`
+
+``:n:`` Any text using the notation syntax (``@id``, ``{+, …}``, etc.).
+    Use this to explain tactic equivalences (for example, you might write this::
+
+       :n:`generalize @term as @ident` is just like :n:`generalize @term`, but
+       it names the introduced hypothesis :token:`ident`.
+
+    Note that this example also uses ``:token:``.  That's because ``ident`` is
+    defined in the the Coq manual as a grammar production, and ``:token:``
+    creates a link to that.  When referring to a placeholder that happens to be
+    a grammar production, ``:token:`…``` is typically preferable to ``:n:`@…```.
+
+``:production:`` A Sphinx role to format grammar productions not included in a
+    `productionlist` directive.
```

Comment:
GitHub doesn't seem to understand this two line explanation. And note that verbatim should take two backquotes.

Reply:
This looks like a comment you missed.

Reply:
Yup,thanks